integration/volume_copy_up_test.go

```diff
@@ -93,7 +93,7 @@ func TestVolumeCopyUp(t *testing.T) {
 			},
 		},
 		{
-			containerPath: "C:/weird_test_dir",
+			containerPath: "/C:/weird_test_dir",
```

Comment:
It seems like we don't need to touch this line?

Reply:
Yes, we do :(

This is the expected volume. We compared the expected with what we got from `getContainerBindVolumes()`. After the change to make the paths absolute, `getContainerBindVolumes()` returns `"/C:/weird_test_dir"`, which doesn't match, and without the change to make the path absolute, the test fails earlier as runc doesn't start the container and throws:

```
    volume_copy_up_test.go:73: Start the container
E0727 12:20:24.801888   14925 remote_runtime.go:270] StartContainer "6e8c79ca7d5fad1defd9fa5352c58835a14f25afa3c2c23a7f082ef1080f6e0d" from runtime service failed: rpc error: code = Unknown desc = failed to create containerd task: failed to create shim task: OCI runtime create failed: runc create failed: invalid mount &{Source:/var/lib/containerd-test/io.containerd.grpc.v1.cri/containers/6e8c79ca7d5fad1defd9fa5352c58835a14f25afa3c2c23a7f082ef1080f6e0d/volumes/d6c535c37b0f3e4599809c8c2b4a99cc6575e7e63a2c6e256e8db8b0a26013e4 Destination:C:/weird_test_dir Device:bind Flags:20480 PropagationFlags:[278528] Data: Relabel: RecAttr:<nil> Extensions:0 UIDMappings:[] GIDMappings:[]}: mount destination not absolute: unknown

```

Reply:
I'm marking it as solved, but if I'm missing something, do not hesitate to re-open it! :)